templates/aframe-model-template/app/assets/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Default scene! • Made with A-Frame</title>
+    <link rel="stylesheet" href="css/app.css">
+    <script src="js/vendor.js"></script>
+    <script src="js/app.js"></script>
+    <script>
+    AFRAME.registerComponent('teleport', {
```

Comment:
but I can see it being useful to be inline so it's easier to read the source from viewing the HTML.

you can also do `<script src="js/components/teleport.js"></script>`, and that'd work